package.json

```diff
@@ -45,21 +45,18 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@energyweb/credential-governance": "^2.1.0",
-    "@energyweb/vc-verification": "^2.1.0",
+    "@energyweb/credential-governance": "^2.2.1-alpha.291.0",
+    "@energyweb/vc-verification": "^2.2.1-alpha.291.0",
     "@ensdomains/ens": "^0.4.5",
     "@ensdomains/resolver": "^0.2.4",
-    "@ethersproject/abstract-provider": "5.4.1",
-    "@ethersproject/abstract-signer": "5.4.1",
-    "@ethersproject/providers": "^5.4.5",
-    "@ew-did-registry/claims": "^0.7.0",
-    "@ew-did-registry/credentials-interface": "^0.7.0",
-    "@ew-did-registry/did-document": "^0.7.0",
-    "@ew-did-registry/did-ethr-resolver": "^0.7.0",
-    "@ew-did-registry/did-ipfs-store": "^0.7.0",
-    "@ew-did-registry/jwt": "^0.7.0",
-    "@ew-did-registry/keys": "^0.7.0",
-    "@ew-did-registry/proxyidentity": "^0.7.0",
+    "@ew-did-registry/claims": "^0.7.1-alpha.777.0",
+    "@ew-did-registry/credentials-interface": "^0.6.3-alpha.729.0",
+    "@ew-did-registry/did-document": "^0.6.3-alpha.729.0",
+    "@ew-did-registry/did-ethr-resolver": "^0.6.3-alpha.729.0",
+    "@ew-did-registry/did-ipfs-store": "^0.7.1-alpha.777.0",
+    "@ew-did-registry/jwt": "^0.6.3-alpha.729.0",
+    "@ew-did-registry/keys": "^0.6.3-alpha.729.0",
```

Comment:
@JGiter Is this a downgrade of some of the ew-did-registry packages?

Reply:
Fixed. Thanks for noticing